roscala/src/main/scala/coop/rchain/rosette/VirtualMachine.scala

```diff
@@ -506,12 +510,15 @@ object VirtualMachine {
 
                 case StoreGlobal(env) => newState.set(_ >> 'globalEnv)(env)
               }
+               */
```

Comment:
Could we delete this code?

Reply:
It's a blue print for the rewrite I'm currently working on. I commented it out because it's not compatible with the rewrite of `Location.scala`.